VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -0,0 +1,147 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
```

Comment:
Still an extra "s", sorry, English is strange...
Should be "Points of Contact" per https://en.wiktionary.org/wiki/points_of_contact#English